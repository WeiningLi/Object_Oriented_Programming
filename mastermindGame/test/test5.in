A
Y
A
Y
A
Y
A
Y
A
Y
A
Y

A
Y
A
Y
C
N
