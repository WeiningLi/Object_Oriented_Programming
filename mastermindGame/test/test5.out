Welcome to Mastermind! Please enter your first guess:
1b,0w
You won in 1 guesses!
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
1b,0w
You won in 1 guesses!
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: E
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: C
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: D
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: B
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
1b,0w
You won in 1 guesses!
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: E
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
0b,0w
You lost! The password was: E
Would you like to play again? (Y/N): 
